### PR TITLE
Feature/develop/task2

### PR DESCRIPTION
**Pull request by shoolak**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+jobs:
+  cpp-arduino-build:
+    runs-on: windows-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      # Забезпечити налаштування Arduino CLI (якщо ще не доступний на ранері)
```
Comment (ljossha) [inline .github/workflows/ci.yml:19]: ?? What happened with encoding?

Comment (shoolak) [reply .github/workflows/ci.yml:19]: have no idea actually
looks normal on my side in the branch

Comment (shoolak) [reply .github/workflows/ci.yml:19]: i can change all comments to english if needed, perhaps there is some issue with ukrainian

Comment (shoolak) [reply .github/workflows/ci.yml:19]: even here it looks normal btw

**Pull request merged**